src/main/kotlin/io/snyk/plugin/services/SnykApplicationSettingsStateService.kt

```diff
@@ -26,7 +26,10 @@ import java.util.UUID
     storages = [Storage("snyk.settings.xml", roamingType = RoamingType.DISABLED)],
 )
 class SnykApplicationSettingsStateService : PersistentStateComponent<SnykApplicationSettingsStateService> {
-    val requiredLsProtocolVersion = 16
+    // events
+    var pluginInstalledSent: Boolean = false
```

Comment:
Will plugin developers working towards prod send a plugin installed event every time we start debug sessions? Will this be an issue for reporting?

Reply:
should not be an issue, as these versions would have a snapshot version of the integrationName.

Reply:
The issue would only occur in our org anyways.